.gitignore

```
@@ -1,3 +1,5 @@
 *.egg-info/
 __pycache__/
 *.ipynb_checkpoints*
+build/*
```

You don't need the glob if you want to ignore an entire directory:
```suggestion
build/
```